packages/storybook/config/storybook/manager-head.html

```diff
@@ -58,6 +58,9 @@
         font-feature-settings: "tnum"; /* Enable tabular numbers */
     }
 
+    #storybook-preview-wrapper {
```

Comment:
If this is set in the manager-head, does it even affect the preview? (Storybook theming is so weird)

Reply:
yes it does weirdly, the base background for preview is set in the manager section, seems odd to me to